### DISCUSSION TITLE
Override account widget

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#8752] [Help] [answered]
i'm currently upgrading from filament version 2 to filament version 3 

in version 2 , i managed to override the account template but after upgrading it doesn't work 
here is the hierarchy i used to override the template 

resources
│
└── views
    │
    └── vendor
        │
        └── filament
            │
            └── widgets
                │
                └── account-widget.blade.php

what should i do ? and is it different in filament v3

**Comment by leandrocfe** [comment]
Put your content here `resources/views/vendor/filament-panels/widgets/account-widget.blade.php`

**Reply by leandrocfe** [reply, marked as answer]
`filament-panels` (folder name) instead of `filament`
